src/CMakeLists.txt

```diff
@@ -131,24 +131,24 @@ set(LocalDependencies ${LocalDependencies} PARENT_SCOPE)
 #     BUNDLE DESTINATION "."
 #     RUNTIME DESTINATION bin)
 # ~~~
-# Using macro from cmake/cleanCpppExtensions.cpp included on the top Provides exampleApp-run and
-# exampleApp-dbg
+# Using macro from cmake/cleanCppExtensions.cpp included on the top Provides run_exampleApp and
+# gdb_exampleApp
 # ~~~
 # add_run_and_debug_targets(exampleApp)
 # ~~~
 #
 # short convenience target from cmake/cleanCpppExtensions.cpp
 # ~~~
-# add_custom_target(run DEPENDS exampleApp-run)
+# add_custom_target(run DEPENDS run_exampleApp)
 # ~~~
 #
 # Manually specified convenience target for running the binary, we use macro
 # ~~~
-# add_custom_target(exampleApp-run
+# add_custom_target(run_exampleApp
 #     WORKING_DIRECTORY ${CMAKE_INSTALL_PREFIX}/bin
 #     USES_TERMINAL
 #     COMMAND ./exampleApp
-#     DEPENDS Install_)
+#     DEPENDS exampleApp)
 # ~~~
 # And this would add -DBAR_ENABLE_FEATURE or equivalent to build command
 # ~~~
```

Comment:
@MathiasPaulin, @nmellado I'm not sure these docs are uptoddate (at least there where not according to target names)
Do anyone uses run_ or gdb_ targets ?
I think we can remove Install_Radium target whose old name was Install_ and used in add_run_and_debug_targets

Reply:
@dlyr OK to remove run_ , gdb_ and add_run_and_debug_targets, I was inclined to do that several times but did not as I though you use them. They come from CleanCppExtensions.cmake script we have used a looong time ago and you should also remove this script from the cmake directory
For Install_Radium (I renamed it from Install_ in this PR) I think we can keep it as it simplifies the usage of some IDE that do not expose the target install in their target menu (VisualStudio, VisualStudioCode, Clion) but only in in their "Build" menu and without keyboard shortcut. We had several experiences of user that do not find how to install Radium and we answered each time to use this target ...
But as I do not have a better reason to keep it, other than "I'm used to it and it simplify my IDE usage",  feel free also to remove it. But it does not implies huge maintenance effort so ...

Reply:
@dlyr You should also remove `ForceRadiumInstallation` custom target (and the few lines above) as well as adding Headless if build in the (useful) target RadiumLibs

Reply:
@dlyr I also think that obsolete (since long time) lines 102 to 199 of src/CMakeLists.txt should be removed